src/components/NavBar/NavBar.css

```diff
@@ -10,6 +10,12 @@
     width: 100%;
     font-size: 16px;
   }
+
+  .location{
+    margin-left:40%;
```

Comment:
A good linter like https://prettier.io/ should be able to pick up the spacing inconsistency here